src/main/java/org/sunbird/workflow/service/impl/RequestServiceImpl.java

```diff
@@ -100,4 +102,33 @@ public Object fetchResultUsingPost(StringBuilder uri, Object request, Class obje
 		return response;
 	}
 
+	public Map<String, Object> fetchResultUsingPatch(String uri, Object request, Map<String, String> headersValues) {
```

Comment:
Check the latest update on the cb-ext service... need to handle exception and return error.
@https://github.com/sunbird-cb/sunbird-cb-ext/tree/juhi-sept-release